types/types_test.go

```diff
@@ -87,8 +88,11 @@ func TestTypeRoundtrip(t *testing.T) {
 				&FixedBinaryType{Nullability: n, Length: 45},
 				&IntervalDayType{Nullability: n, Precision: 5},
 				&IntervalDayType{Nullability: n, Precision: 0},
+				compound,
```

Comment:
do we really need the local variable for this?

Reply:
Not needed, originally added to get the pointer, but apparently IntervalCompound needs to be real object.